contrib/guix/README.md

```diff
@@ -9,79 +11,835 @@ downloads.
 
 We achieve bootstrappability by using Guix as a functional package manager.
 
-## Requirements
+# Requirements
 
 Conservatively, a x86_64 machine with:
 
 - 16GB of free disk space on the partition that /gnu/store will reside in
 - 8GB of free disk space per platform triple you're planning on building (see
   the `HOSTS` environment variable description)
 
-## Setup
+# Installation and Setup
+
+At the time of writing, the latest Guix release is v1.2.0, which has a known
```

Comment:
In 5ae2760f614c0872a2289574477543070243e236 "guix: Overhaul README"

Latest release is 1.3.0 which fixes the GnuTLS issue, so I think this paragraph is no longer needed.